package.json

```diff
@@ -29,6 +29,7 @@
     }
   },
   "dependencies": {
+    "@carbon/react": "^1.13.0",
```

Comment:
Why introduce a new dependency?

Reply:
Just moved this up from `devDependencies`

Reply:
Yes, I get that part, Carbon is meant to be a singleton dependency. Introducing the dependency above violets that.